### PR TITLE
Enable fio test using containerd client

### PR DESCRIPTION
**Pull request by dborquez**
This PR enables FIO benchmark using containerd in order to measure kata containers IO latency.
The previous version of FIO on k8s was deprecated.

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

File: tests/metrics/storage/fio-dockerfile/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Set up an Ubuntu image with 'fio io tester' installed
+
+# Usage: FROM [image name]
+# hadolint ignore=DL3007
```
Comment (cmaf) [inline tests/metrics/storage/fio-dockerfile/Dockerfile:8]: @dborquez should this be commented out?

**Review by cmaf — APPROVED** [review]

**Pull request merged**